src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer.cs

```diff
@@ -88,7 +88,11 @@ internal static CloudQueueMessage ToCloudQueueMessage<T>(Guid streamGuid, String
         {
             var azureQueueBatchMessage = new AzureQueueBatchContainerV2(streamGuid, streamNamespace, events.Cast<object>().ToList(), requestContext);
             var rawBytes = SerializationManager.SerializeToByteArray(azureQueueBatchMessage);
-            return new CloudQueueMessage(rawBytes);
+
+            //new CloudQueueMessage(byte[]) not supported in netstandard, taking a detour to set it
+            var cloudQueueMessage = new CloudQueueMessage(null as string);
+            cloudQueueMessage.SetMessageContent(rawBytes);
+            return cloudQueueMessage;
```

Comment:
I'm also blaming "the hack" for the runtime exceptions while trying to port codegen - hopefully @attilah's change fixes everything and restores my faith in .NET Standard